assets/pages/contributions-landing/contributionsLanding.jsx

```diff
@@ -20,7 +19,11 @@ import { createPageReducerFor } from './contributionsLandingReducer';
 
 const countryGroupId: CountryGroupId = detect();
 
-const store = pageInit(createPageReducerFor(countryGroupId));
+const store = pageInit(
+  createPageReducerFor(countryGroupId),
+  false,
```

Comment:
I'm not a fan of passing through pure boolean values to functions, and they feel especially weird in the middle. 

I know the alternatives to this are neither bountiful nor particularly desirable, but maybe you could at least put the boolean to the end?

I don't know, this might be a weird preference I have that no one else has, and feel completely free to ignore.

Reply:
does JS allow named parameters? that would help.

Reply:
agreed. JS doesn't have named parameters, except through the "trick" of passing an object instead (you can do object destructuring inside the actual parameter list to get individual variables inside the function body)

going to leave this one since re-ordering args to `pageInit` would affect a lot of places